_posts/2022-05-09-Rails-7-0-3-6-1-6-6-0-5-and-5-2-8-have-been-released.markdown

```diff
@@ -21,7 +21,14 @@ We recommend upgrading to a newer version as soon as possible.
 Only gems with changes in the CHANGELOGs are listed below. All include a bugfix
 for https://github.com/rails/rails/issues/45014 and https://github.com/rails/rails/issues/44972.
 
-### 7.0.2
+To see a summary of changes, please read the release on GitHub:
+
+* [7.0.3 CHANGELOG](https://github.com/rails/rails/releases/tag/v7.0.3)
+* [6.1.6 CHANGELOG](https://github.com/rails/rails/releases/tag/v6.1.6)
+* [6.0.5 CHANGELOG](https://github.com/rails/rails/releases/tag/v6.0.5)
+* [5.2.8 CHANGELOG](https://github.com/rails/rails/releases/tag/v5.2.8)
```

Comment:
I noticed this tags don't have release notes.  Is that expected?  If so, I can just remove these links.

Reply:
Er, I forgot to add them and not sure it can be done after the fact. I'm actually not sure how github release notes work.

Reply:
I can do it. I'm the one that is being doing every release we forget to do.